lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -25,7 +25,10 @@
 #include "torch-mlir/Dialect/Torch/Utils/TorchUpstream.h"
 #include "torch-mlir/Dialect/Torch/Utils/Utils.h"
 #include "llvm/ADT/APSInt.h"
+#include <cassert>
+#include <iostream>
```

Comment:
apparently including `<iostream>` is one of the cardinal sins in a LLVM project file.

Why is this included and can you do the same with just the llvm::errs etc streams?

Reply:
https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden

Reply:
(looking further it looks like you're moving some code from one file to another)

> 
> [Use raw_ostream](https://llvm.org/docs/CodingStandards.html#id52)
> LLVM includes a lightweight, simple, and efficient stream implementation in llvm/Support/raw_ostream.h, which provides all of the common features of std::ostream. All new code should use raw_ostream instead of ostream.
> 
> Unlike std::ostream, raw_ostream is not a template and can be forward declared as class raw_ostream. Public headers should generally not include the raw_ostream header, but use forward declarations and constant references to raw_ostream instances.
-- https://llvm.org/docs/CodingStandards.html#use-raw-ostream

Since we're already touching this code, let's modernize this part

Reply:
embarrassing... it's just left over from some "debugging"